src/components/marker-chart/index.js

```diff
@@ -2,14 +2,17 @@
 import * as React from 'react';
 import { connect } from 'react-redux';
 import MarkerChartCanvas from './Canvas';
+import MarkerTooltipContents from '../shared/MarkerTooltipContents';
 import {
   selectedThreadSelectors,
   getDisplayRange,
   getProfileInterval,
   getProfileViewOptions,
 } from '../../reducers/profile-view';
 import { updateProfileSelection } from '../../actions/profile-view';
+import { getImplementationFilter } from '../../reducers/url-state';
 
+import type { Element as ReactElement } from 'react';
```

Comment:
nit: you can just use `React.Element` as we use `import * as React from 'react'` above.